packaging/neo4j-desktop/src/main/java/org/neo4j/desktop/config/osx/DarwinInstallation.java

```diff
@@ -48,25 +49,25 @@ public DarwinInstallation()
                     "echo neo4j-import",
                     "bash"};
 
-            try ( FileWriter fileWriter = new FileWriter( new File( filename ), false ) )
+            try( FileWriter fileWriter = new FileWriter( new File( filename ), false ) )
```

Comment:
This would probably be simpler using `java.nio.file.Files#write(Path, Iterable<String>)` method.